src/applications/personalization/preferences/tests/components/PreferenceOption.unit.spec.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+import PreferenceOption from '../../components/PreferenceOption';
+import { benefitChoices } from '../../helpers';
+
+const onChange = spy();
+
+const props = {
+  item: benefitChoices[0],
+  onChange,
+  checked: true,
+};
+
+describe('<PreferenceOption>', () => {
+  it('should render', () => {
+    const component = shallow(<PreferenceOption {...props} />);
+    expect(component.debug()).to
```

Comment:
I think this is the first time I've seen component tests that look for specific markup like this. And it feels pretty fragile. Maybe in this case we could just check to make sure that the data from the passed-in `item` appears in the rendered component? Just a thought!

Reply:
I should have added a comment that this is an experiment to simulate jest snapshot testing. I will replace with more conventional tests.